### PR TITLE
ddl: improve the log messages and refine code

### PR DESCRIPTION
**Pull request by tangenta**
### What problem does this PR solve?

Problem Summary:

- Some places don't have a log even if an error is returned.
- `int(variable.GetDDLReorgBatchSize())*copReadBatchFactor` occurs for several times. It could be encapsulated.
- Some of the errors are incorrectly tagged with "[ddl-ingest]" label.
- Some messages are unused.

### What is changed and how it works?

As the title said.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: ddl/ingest/backend.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ type BackendContext struct {
 func (bc *BackendContext) FinishImport(indexID int64, unique bool, tbl table.Table) error {
 	ei, exist := bc.EngMgr.Load(indexID)
 	if !exist {
-		return errors.New(LitErrGetEngineFail)
+		return errors.New("ingest engine not found")
```
Comment (Benjamin2037) [inline ddl/ingest/backend.go:48]: Why directly use err string here?

**Review by wjhuang2016 — APPROVED** [review]

**Review by Benjamin2037 — APPROVED** [review]
LGTM

**Pull request merged**